Adding permissions for shipped languages (#1206)
mattermost/mattermost-handbook: contributors/join-us/localization.md

![image](https://user-images.githubusercontent.com/1832946/227045392-ca3640bd-d0c1-48d2-86ee-657b40e068b2.png)

### Step 3 - Give translators access to your project

> Access for the shipped languages is restricted to approved users. For each language there is a group which yo have to give access to your project.

Reviewer: ```suggestion
Access for the shipped languages is restricted to approved users. You must give access to your project on a per-language basis.
```
